src/g_canvas.c

```diff
@@ -942,11 +942,27 @@ static void canvas_drawlines(t_canvas *x)
         linetraverser_start(&t, x);
         while ((oc = linetraverser_next(&t)))
         {
+            int issignal = outlet_getsymbol(t.tr_outlet) == &s_signal;
+            int nchannels = obj_sigoutletgetchans(t.tr_ob, t.tr_outno);
+            int connection_thickness;
```

Comment:
`connection_thickness` is a bit of a mouthful. I think in this context `width` would be perfectly clear :)